test/controllers/DeclarationControllerSpec.scala

```diff
@@ -150,5 +161,26 @@ class DeclarationControllerSpec extends SpecBase with BeforeAndAfterEach with Ch
       verify(mockCharitiesRegistrationService, never()).register(any())(any(), any(), any())
     }
 
+    "redirect to Tasklist for a GET if SectionPage is not completed" in {
+
+      when(mockUserAnswerRepository.get(any())).thenReturn(Future.successful(Some(emptyUserAnswers
+        .set(Section1Page, false)
```

Comment:
No need to set all Section page values here... just set one for true and one for false